code/datums/armor.dm

```diff
@@ -1,5 +1,18 @@
 #define ARMORID "armor-[melee]-[bullet]-[laser]-[energy]-[bomb]-[bio]-[rad]-[fire]-[acid]-[magic]-[wound]"
 
+#define MELEE "melee"
+#define BULLET "bullet"
+#define LASER "laser"
+#define ENERGY "energy"
+#define BOMB "bomb"
+#define BIO "bio"
+#define RAD "rad"
+#define FIRE "fire"
+#define ACID "acid"
+#define MAGIC "magic"
+#define WOUND "wound"
+
```

Comment:
Could be worth replacing the occurances in other places aswell since you already added these defines. Although not neccessarily in this PR.

Reply:
That will be a pain but i think i'll look into it.

Reply:
I hated it.